test/test-lex.cpp

```diff
@@ -1876,6 +1876,18 @@ TEST_F(test_lex, lex_not_shebang) {
                               offsets_matcher(&input, 2, 3))));
   }
 
+  // BOM must not appear before '#!'.
+  {
```

Comment:
Also test BOM without `#!`. (It should be ignored, producing no error.)

Reply:
I guess there it already a test for that here - https://github.com/quick-lint/quick-lint-js/blob/6c54dc042d691bbbe6d0c83115b5cd6e907c92bd/test/test-lex.cpp#L1836

Reply:
I meant test `\ufeffStuff`, not `#!Stuff`.

Reply:
oh okay, I misread that.

Reply:
> I meant test `\ufeffStuff`, not `#!Stuff`.

Isn't this function testing that already - https://github.com/quick-lint/quick-lint-js/blob/6c54dc042d691bbbe6d0c83115b5cd6e907c92bd/test/test-lex.cpp#L1781

Let me know if I'm wrong.